test/TestAssets/NuGet.config

```diff
@@ -5,10 +5,28 @@
   </packageRestore>
   <packageSources>
     <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
```

Comment:
Are these changes linked to some rebase/merge or have you done some manual changes?

Reply:
Copy pasted from the real nuget config, I forgot to remove the comments. I think we have 2 because the one for assets is edited by the build script.

Reply:
removed